install.sh

```diff
@@ -5,14 +5,18 @@ set -e
 osName=$(uname -s)
 
 osArchitecture=$(uname -m)
-if [[ $osArchitecture == *'aarch'* || $osArchitecture == *'arm'* ]]; then
+
+if [ $osName == "Darwin" ]; then
```

Comment:
We need to make this release to only work for `M1` arch and `x86_64` to avoid installing this executable for any other architectures